scripts/pyro_name_server.py

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env python
+"""
+Script to run the Pyro name server. This must be running in order to use distributed cameras with
+POCS. The name server can be run on any computer on the network, but would normally be run on the
+main control computer. The name server should be started before the distributed camera servers and
+POCS.
+"""
+import argparse
+
+from huntsman.utils.pyro import run_name_server
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--host", help="hostname or IP address to bind the server on")
+    parser.add_argument("--port", help="port number to bind the server on")
+    parser.add_argument("--autoclean", help="interval between registration autoclean (0=disabled)",
```

Comment:
Did we want to enable this by default or do you think not needed?

Reply:
I think it should be enabled by default.  With the `finally` in the camera server they will now de-register themselves from the name server if you shut them down, but that doesn't handle more messy situations (network problems, Pi power failure, out of memory errors, etc.).  The name server autoclean is a handy catch-all backup.